hwy/ops/arm_neon-inl.h

```diff
@@ -5719,6 +5757,27 @@ HWY_INLINE Vec128<int64_t> SumOfLanes(hwy::SizeTag<8> /* tag */,
   return v + Shuffle01(v);
 }
 
+HWY_INLINE uint32_t SumOfLanesVal(hwy::SizeTag<4> /* tag */,
```

Comment:
Also, would it make sense to implement SumOfLanes in terms of 'scalar SumOfLanes' rather than the other way around? That way you avoid a redundant Set/Get.